modules/account-lib/src/coin/celo/transactionBuilder.ts

```diff
@@ -89,7 +89,7 @@ export class TransactionBuilder extends Eth.TransactionBuilder {
     return walletSimpleByteCode + resultEncodedParameters;
   }
 
-  //region Stake methods
+  // region Stake methods

```

Comment:
@twtaylorbitgo says this isn't supposed to be here but I dunno :shrug:

Reply:
Do you mean it should be in another line, or we should not add the region at all? @twtaylorbitgo

Reply:
Resolved this via private conversation